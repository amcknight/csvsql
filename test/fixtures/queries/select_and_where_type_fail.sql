SELECT NOT age AS notAge
FROM test/fixtures/tables/people
WHERE NOT age
