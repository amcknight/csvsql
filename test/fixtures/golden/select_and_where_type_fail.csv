Type Error in Not (Var people.age)
              ^^^^^^^^^^^^^^^^^^^^
Expected Boolean but found Integer